.vscodeignore

```diff
@@ -1,23 +1,24 @@
-.vscode
-.vscode-test
+.github/
+.husky/
+.vscode/
+.vscode-test/
 out/
+scripts/
 src/
 test-fixtures/
-tsconfig.json
-webpack.config.js
-.prettierignore
-azure-pipelines.yml
-scripts/
-samples/
+.gitattributes
 .gitignore
-.github/
+.nvmrc
+.prettierignore
+.prettierrc
+CODE_OF_CONDUCT.md
+CONTRIBUTING.md
+CONTRIBUTORS.md
 tsconfig.json
+webpack.config.js
```

Comment:
why? :)

Reply:
Can you clarify? If you mean why is the `webpack.config.js` there, then that would be because it shouldn't be included in the final bundle (and it was already there before this PR).